pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -71,6 +75,18 @@ const (
 )
 
 const (
+	cephOSDStart = `
+%s %s
+
+RESTART_INTERVAL=%.2f
```

Comment:
Is this interval not an int?

Reply:
No. `.Hours()` response is float. Can't convert to int, as it will return the `floor` value.

Reply:
casting to `int` won't work if the use gives a custom restart interval in minutes, lets say `30m`.  Converting this interval to `hours` and then casting to int would return 0. 

We can cast to int if we don't expect user to give interval in minutes.

Reply:
If we need to support minutes, then let's always convert to minutes and use the suffix `m`. Keeping it an integer is just better IMO. Are you using minutes for private testing? Not sure why a user would want something less than an hour though.

Reply:
You are right. It won't make much sense to wait for `minutes` we should only wait for hours. 
Yes, I was testing minutes in private testing.